Resources/public/css/theme/views/studiopresentation.css

```diff
@@ -4,7 +4,10 @@
  */
 
 .ez-view-studiopresentationview {
-    /* for test purpose */
-    padding: 1em;
+    padding: 0px;
     background: #fff;
 }
+
+.ez-view-studiopresentationview iframe{
```

Comment:
usually we avoid styling based on the tag name, so you should add a class on the iframe (.ez-studiopresentation-content for instance) and use it here.
CS nitpick: space before `{`
nitpick 2: you don't need to add the unit when setting something to 0.